lib/smart_answer/formatting_helper.rb

```diff
@@ -2,12 +2,18 @@ module SmartAnswer
   module FormattingHelper
     include ActionView::Helpers::NumberHelper
 
-    def format_money(amount)
+    def format_money(amount, pounds_only: false)
       amount = extract_number(amount)
       if show_in_pence?(amount)
         number_to_currency(amount * 100, precision: 0, unit: 'p', format: '%n%u')
       else
-        number_to_currency(amount, precision: ((amount == amount.round) ? 0 : 2))
+        precision =
+          if pounds_only || amount == amount.round
+            0
+          else
+            2
+          end
```

Comment:
@h-lame do you think a ternary operator would not be readable enough here?

Reply:
I think when the condition is more than a single clause it can get confusing to read (as you have to start thinking about operator precedence.  My preference was to add brackets, but rubocop didn't like it when I put brackets around the ternary so I exploded it out into a full if statement.

Reply:
Rejigged this after a discussion on slack.